site/src/entries/chemical-extract.md

```diff
@@ -8,12 +8,23 @@ TODO intro paragraph and how to begin to add
 
 ## Required Information
 
-- TODO
+- **Nickname:** When you have several Isolates, it may be useful to give them short names to help distinguish them. Please follow your instructor's guidelines on how to name entries meaningfully.
+- **Date:** Date: The date the isolate was first created.
+
 
 ## Optional Information
 
-- TODO
+- **Isolate:** Click the "add Isolate" button to link an isolate to this entry. This field is automatically populated for you if you click "Add Eukaryotic Test" to an existing isolate.
+- **Media Used and Concentration of Media Used:** The base media used to cultivate the plate and its concentration. For example, commonly when the Media Used is TSA, the Concentration is 10%. Confirm with your instructor the concentration of your media.
```

Comment:
Instead of:
`**Media Used and Concentration of Media Used:**`

Do:
`**Media Used** and **Concentration of Media Used:**`

This way the "and" isn't bolded too, making it clearer that this is one line that refers to two fields

Reply:
Same with Additives